owncloudApp/src/main/java/com/owncloud/android/providers/WorkManagerProvider.kt

```diff
@@ -76,4 +81,13 @@ class WorkManagerProvider(
         WorkManager.getInstance(context)
             .enqueueUniquePeriodicWork(AVAILABLE_OFFLINE_PERIODIC_WORKER, ExistingPeriodicWorkPolicy.KEEP, availableOfflinePeriodicWorker)
     }
+
+    fun getWorkInfoByTagsLiveData(): LiveData<List<WorkInfo>> {
```

Comment:
This function chain is a little bit weird from my point of view. I think we could directly call getWorkInfosLiveData building the expected workQuery with the expected tags and status (`buildWorkQuery`) and rename it to a self-explanatory name. What do you think?
```suggestion
    fun getOnGoingUploadsWorkInfosLiveData(): LiveData<List<WorkInfo>> {
```

Reply:
Well, we already have the `getRunningWorkInfosByTagsLiveData` extension for WorkManager, so I think it can be useful in this case. The provider makes it easy to mock it in tests. However, I agree with the renaming 😀